core/src/test/scala/org/apache/spark/sql/delta/DeltaProtocolVersionSuite.scala

```diff
@@ -247,7 +247,7 @@ trait DeltaProtocolVersionSuiteBase extends QueryTest
 
     testTableCreation { dir =>
       spark.range(10).write.mode("append").parquet(dir)
-      sql(s"CONVERT TO DELTA parquet.`$dir`")
+      sql(s"CONVERT TO DELTA parquet.`$dir` NO STATISTICS")
```

Comment:
same here. why this change?

Reply:
Since eventually we convert to delta with stats on a single commit, there is no need for that to make the tests pass. Removed.